core/tests/protractor_utils/users.js

```diff
@@ -107,49 +73,48 @@ var _completeSignup = async function(username, manualNavigation = true) {
   await waitFor.pageToFullyLoad();
 };
 
-var completeLoginFlowFromStoryViewerPage = async function(email, username) {
-  await _login(email, false, false);
-  await _completeSignup(username, false);
+var createAndLoginUser = async function(email, username) {
+  await _createFirebaseAccount(email);
+  await login(email);
+  await _completeSignup(username);
 };
 
-var createUser = async function(email, username) {
-  await createAndLoginUser(email, username);
-  await logout();
+var createAndLoginAdminUser = async function(email, username) {
+  await _createFirebaseAccount(email, true);
+  await login(email);
+  await _completeSignup(username);
+  await adminPage.get();
+  await adminPage.updateRole(username, 'admin');
 };
 
-var createAndLoginUser = async function(email, username) {
+var createAndLoginAdminUserMobile = async function(email, username) {
+  await _createFirebaseAccount(email, true);
   await login(email);
   await _completeSignup(username);
 };
```

Comment:
It looks like the difference between `createAndLoginAdminUser` and `createAndLoginAdminUserMobile` is that `createAndLoginAdminUser` assigns the user the "admin" role while `createAndLoginAdminuserMobile` does not. In that case, wouldn't the user from `createAndLoginAdminUserMobile` not be an admin?

Reply:
I don't know, I'm merely copying pre-existing behavior. This was changed in #5340 (https://github.com/oppia/oppia/pull/5340/files#diff-218a7c7ead12cb4e3bf758923d86edaf5390a33ae9ef07e13acb99101bba768e) without any explanation, so I guess we can remove this in a future PR.

Reply:
OK, thanks for investigating!